addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -193,7 +193,7 @@
                             </div>
                         </tr>
                         <tr>
-                            <div t-if="task.total_hours_spent > 0 and timesheets_by_subtask" name="total_time">
```

Comment:
Updating the 'if' condition will cause the customization to break.

What do you consider?

Reply:
The risk is very low if we alter the `if` condition since the xpath in the customization could select `name` attribute.